### PR TITLE
UI Changes: Updated search bar for explore pages

### PR DESCRIPTION
**Pull request by juliawu**
Updates the styling of our search bars on the explore pages. Also removes the `experimental` tag in favor of "Early Preview" text.

Screenshots:
Screenshot 2023-08-15 at 5 06 43 PM

Screenshot 2023-08-15 at 5 14 42 PM

### CODE REVIEW

File: static/css/explore.scss [hunk 1]
```diff
@@ -66,26 +66,31 @@
...
-    color: var(--blue);
-    display: flex;
-    float: left;
-    font: normal 0.85em monospace;
-    padding: 4px 6px;
-    height: 80%;
-    margin-right: 20px;
+  .early-preview-tag {
+    color: var(--gm-3-ref-secondary-secondary-30, #004a77);
+    font-family: Google Sans;
```
Comment (juliawu) [inline static/css/explore.scss:83]: Reviewers: Using "Google Sans" throughout for now until base.scss can be updated. Let me know if there's a different value to prefer or if I should remove font-family entirely.

File: static/js/components/text_search_bar.tsx [hunk 2]
```diff
@@ -51,12 +51,25 @@ export function TextSearchBar(props: TextSearchBarPropType): JSX.Element {
...
-          className="pac-target-input"
+          className="pac-target-input search-input-text"
           autoFocus={props.shouldAutoFocus}
-        />
-        <Button onClick={handleSearch} id="rich-search-button">
-          <span className="material-icons search rich-search-icon">search</span>
-        </Button>
+          autoComplete="off"
+        ></Input>
+        <div onClick={handleSearch} id="rich-search-button">
```
Comment (pradh) [inline static/js/components/text_search_bar.tsx:58]: nit: add some comment on the source of this?

**Review by pradh — APPROVED** [review]
Very nice!

File: static/js/components/text_search_bar.tsx (hunk 2 above)
Comment (juliawu) [reply static/js/components/text_search_bar.tsx:58]: What do you mean by "this"?

Comment (pradh) [reply static/js/components/text_search_bar.tsx:58]: I mean, this blob of SVG, where did you come up with its content?

Comment (juliawu) [reply static/js/components/text_search_bar.tsx:58]: Ah, took that from the figma mocks from Ido and Adriana, should I still add a note?

Comment (pradh) [reply static/js/components/text_search_bar.tsx:58]: ok, nm

**Comment by pradh** [comment]
Sorry, merged this in to get this into autopush.  Real reviewers, PTAL anyway :)

File: static/js/components/text_search_bar.tsx (hunk 2 above)
Comment (dwnoble) [reply static/js/components/text_search_bar.tsx:58]: I'd store this as `search-arrow.svg` in the images folder and include it here with an ``

Comment (juliawu) [reply static/js/components/text_search_bar.tsx:58]: Ok, will send followup PR to cleanup the svg!

**Pull request merged**